lib/manager.dart

```diff
@@ -115,20 +99,35 @@ class _Manager extends State<Manager> {
     });
   }
 
+  // void _setKaraokeButton(String path) {
+  //   if(path == 'sdcard/default.m4a'){
+  //     setState(() {
+  //      _karaokeButton = 'assets/stop.jpeg'; 
+  //     });
+  //   } else if(path == 'stopped player.') {
+  //     setState(() {
+  //       _karaokeButton = 'assets/start.jpeg'; 
+  //     });
+  //   }
+  // }

```

Comment:
what about this commented out code? :) what is the reason you decided to keep it?

Reply:
This entire code works so I think ok for merge.
However I still try on change button state between startAudio and stopAudio.